src/workerd/api/http.c++

```diff
@@ -1231,6 +1235,31 @@ kj::Promise<DeferredProxy<void>> Response::send(
         "Worker tried to return a WebSocket in a response to a request "
         "which did not contain the header \"Upgrade: websocket\".");
 
+    if (outHeaders.get(kj::HttpHeaderId::SEC_WEBSOCKET_EXTENSIONS) == nullptr) {
```

Comment:
Maybe this whole block should only run if `canUseWebSocketCompression()` is true?

Reply:
https://github.com/cloudflare/workerd/pull/69/commits/f3dc65275652255f24561af8dbabf9d1f56ec0bb

I think we still need need to check if compression is enabled and `unset` if it's not after this block. Ex. consider a worker script that explicitly includes the `Sec-WebSocket-Extensions` header in a response when compression is not enabled. Will add that back in now.